app/angular/recovery/recovery.js

```diff
@@ -13,23 +13,23 @@ const controller = function (AuthService) {
 	};
 
 	const showFeedback = (newMessage, type = "error") => {
-		ctrl.feedback.message = newMessage;
+		ctrl.feedback.message = $filter('translate')(newMessage);
 		ctrl.feedback.showing = true;
 		ctrl.feedback.type = type;
 	};
 
 	const handleError = (error) => {
 		ctrl.loading = false;
 		if(error.status == 400) {
-			showFeedback("Esse email ainda não está cadastrado.");
+			showFeedback("This email is not registered yet.");
 		} else {
-			showFeedback("Ops, tivemos um erro interno, por favor tente novamente mais tarde...");
+			showFeedback("Ops, we had an internal error, please try again later...");
 		}
 	};
 
 	const handleSuccess = () => {
 		ctrl.loading = false;
-		showFeedback(`Um email com instruções foi enviado para ${ctrl.mail}`, "success");
+		showFeedback($filter('translate')('An email with instructions has been sent to X', { email: ctrl.mail }), "success");
```

Comment:
```suggestion
		showFeedback($filter('translate')('An email with instructions has been sent to EMAIL', { email: ctrl.mail }), "success");
```

Again, I think we could have a meaningful word instead of just `X`